### ISSUE TITLE
Open in Safari (from Firefox) issue 

### ISSUE THREAD

**Issue by Rackmani** [andy-portmen/open-in#51]
Hi Andy et al,

Love this extension  using in Firefox (current v 84.0.1) on a Mac, latest ios Big Sur 11.1.  Have just noted that this extension still works perfectly  when Safari is already open, but if not open it does still open the page as a new tab in Safari as before, but also opens that web page as a new repeat tab in Firefox which it goes to.

It didn't used to do this.  Can you fix? 

https://user-images.githubusercontent.com/57327107/103459038-29b70400-4d04-11eb-8e06-baae167b34ec.mov

**Comment by andy-portmen** [comment]
True, as far as I can tell, the problem is not with this extension. I guess this is a bug with macOS Big Sur.

Even running the `open -a safari http://example.com` from the terminal when Safari is not open, opens a copy in the default browser. I even tried to open a link with apple script and got the same behavior

```applescript
tell application "Safari"
    open location "http://example.com"
    activate
end tell
```

**Comment by Rackmani** [comment]
Thanks Andy for looking into this.  Is it worth reporting?  And if so to whom/where do I report it?

**Comment by andy-portmen** [comment]
These sorts of issues normally get fixed by OS updates. Anyhow you may want to report to https://discussions.apple.com/
